mkosi/__init__.py

```diff
@@ -4126,14 +4127,18 @@ def secure_boot_sign(
         return
 
     with mount():
-        for path, _, filenames in os.walk(root / "efi"):
+        base_path = root / "efi" if esp else root / "usr/lib/systemd/boot/efi"
+        for path, _, filenames in os.walk(base_path):
```

Comment:
Since `root` comes in as a `Path` nowadays, this could be a `.glob("*.efi")` (which unfortunately would need to be chained with `.glob("*.EFI")`) .

Reply:
I can't say I fully understand. Can you give me a short code snippet as an example?

Reply:
```diff
-       for path, _, filenames in os.walk(base_path):
+       for path in itertools.chain(base_path.glob("*.efi"), base_path.glob("*.EFI")):
```